cmd/diagnostic_test.go

```diff
@@ -111,69 +107,52 @@ token: mytoken
 				session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
 				Expect(err).ShouldNot(HaveOccurred())
 				Eventually(session.Err.Contents()).Should(BeEmpty())
-				Eventually(session.Out).Should(gbytes.Say("GraphQL API address: https://example.com/graphql"))
+				Eventually(session.Out).Should(gbytes.Say(
+					fmt.Sprintf("GraphQL API address: %s", testServer.URL())))
 				Eventually(session.Out).Should(gbytes.Say("OK, got a token."))
 				Eventually(session).Should(gexec.Exit(0))
 			})
 		})
 
-		Describe("token, host, and new endpoint set in config file", func() {
```

Comment:
The original version of this test is basically moved here, right? https://github.com/CircleCI-Public/circleci-cli/blob/32fd81e7cffa37fa16ae362351a46f0de91eb9a6/api/api_test.go#L8-L45

Reply:
@eric-hu More or less, this test is covered here:

```
 	addr, _ = GraphQLServerAddress("graphql-unstable", "https://example.com") 
 	expected = "https://example.com/graphql-unstable" 
 	if addr != expected { 
 		t.Errorf("Expected %s, got %s", expected, addr) 
 	} 
```